test/parallel/test-process-release.js

```diff
@@ -18,3 +18,19 @@ if (versionParts[0] === '4' && versionParts[1] >= 2) {
 } else {
   assert.strictEqual(process.release.lts, undefined);
 }
+
+const {
+  majorVersion: major,
+  minorVersion: minor,
+  patchVersion: patch,
+  prereleaseTag: tag,
+  compareVersion,
+} = process.release;
+
+assert.strictEqual(0, compareVersion(major, minor, patch, tag));
```

Comment:
Can you change the order of the arguments in these assertions to reflect the `assert.strictEqual(actual, expected)` signature.